qutebrowser/commands/userscripts.py

```diff
@@ -344,6 +344,14 @@ def run(cmd, *args, win_id, env, verbose=False):
     if user_agent is not None:
         env['QUTE_USER_AGENT'] = user_agent
     cmd = os.path.expanduser(cmd)
+
+    # check if userscript is in absolute path or in $XDG_DATA_DIRS
+    if not os.path.isabs(cmd):
+        log.misc.debug("{} is no absoulte path".format(cmd))
+        c = os.path.join(standarddir.data(), "userscripts", cmd)
+        if os.path.isfile(c):
+            cmd = c
```

Comment:
I don't think the check is needed - if the file doesn't exist, it'll just trigger an error later and show that to the user, no?